cmds/core/lockmsrs/lockmsrs.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2012-2020 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// lock-msrs locks a bunch of intel MSRs, all specified in the Intel
```

Comment:
Try to make this a one-liner followed by a description section. This will make it look nice on the godoc cmds page: https://godoc.org/github.com/u-root/u-root

Reply:
without dash, right?